src/DifferentThemeVariant.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Phpab\Phpab;
+
+class DifferentThemeVariant implements VariantInterface
```

Comment:
No this will not be part of the lib. It's just an example.
By default only CallbackVariant should be implemented (at the beginning)